src/components/common/text-input.tsx

```diff
@@ -23,3 +24,37 @@ export const TextInput: FunctionComponent<ButtonProp> = ({ className, placeHolde
     />
   );
 };
+
+export const DataList: FunctionComponent<ButtonProp> = ({
+  className,
+  placeHolder,
+  onChange,
+  value,
+  options,
+  dataTestId,
+}) => {
+  const _onChange: (event: ChangeEvent<HTMLInputElement>) => void = ({ target }) => {
+    return onChange(target.value === "" ? "" : target.value);
+  };
+
+  return (
+    <>
+      <input
+        className={`${className} py-2.5 px-5 border-2 border-gray-600 rounded `}
+        type="text"
+        list="data"
+        onChange={_onChange}
+        value={value}
+        placeholder={placeHolder}
+        data-testid={dataTestId}
+      />
+      {options != undefined && (
+        <datalist id="data">
```

Comment:
![image](https://user-images.githubusercontent.com/503185/125019815-3d279b00-e0aa-11eb-8c19-2af9e04f5970.png)

Ugly

Reply:
Fixed, change to react-select. the style look better